kong/templates/nginx_kong.lua

```diff
@@ -91,6 +91,11 @@ server {
     ssl_certificate_by_lua_block {
         kong.ssl_certificate()
     }
+
+    ssl_session_cache shared:SSL:10m;
+    ssl_session_timeout 10m;
+    ssl_prefer_server_ciphers on;
+    ssl_ciphers ${{SSL_CIPHERS}};
```

Comment:
This either needs to be added to the Admin server as well, or the property should reflect that those values are for the Proxy port only (the former is probably what we are after here).